packages/components/src/internal/app/utils.ts

```diff
@@ -375,10 +409,30 @@ export function isProtectedDataEnabled(moduleContext?: ModuleContext): boolean {
     return hasModule('compliance', moduleContext) && hasModule('complianceActivities', moduleContext);
 }
 
+export function isNamingPrefixEnabled(moduleContext?: ModuleContext): boolean {
+    return isBiologicsEnabled(moduleContext);
+}
+
 export function isMediaEnabled(moduleContext?: ModuleContext): boolean {
     return isFeatureEnabled(ProductFeature.Media, moduleContext);
 }
 
+export function isRegistryEnabled(moduleContext?: ModuleContext): boolean {
+    return isBiologicsEnabled(moduleContext) && isFeatureEnabled(ProductFeature.BiologicsRegistry, moduleContext);
```

Comment:
why not just have this check for the `BiologicsRegistry` feature enabled? I assume that feature would never be enabled without the `isBiologicsEnabled` being true

Reply:
If you have your product configuration set to Biologics Enterprise, then if you just check for the feature flag, when you look at Sample Manager application, it will have registry-like qualities.  This is a substitute in some places for what used to be `isBIologicsPrimaryApp`.